versions.gradle

```diff
@@ -21,7 +21,7 @@
 * maven repositories.
 **/
 ext.deps = [:]
-def versions = [:]
+ext.versions = [:]
```

Comment:
I think we would like to put back `def versions = [:]` and add `ext.versions = versions` after Ln 52.

Reply:
Fixed